packages/flutter_goldens/lib/src/flutter_goldens_web.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async' show FutureOr;
+import 'dart:convert' show json;
+import 'dart:html' as html;
+
+import 'package:flutter_test/flutter_test.dart';
+
+import 'flaky_goldens.dart';
+
+export 'package:flutter_goldens_client/skia_client.dart';
+
+/// Wraps a web test, supplying a custom comparator that supports flaky goldens.
+Future<void> testExecutable(FutureOr<void> Function() testMain, {String? namePrefix}) async {
+  webGoldenComparator = FlutterWebGoldenComparator(webTestUri);
+  await testMain();
+}
+
+/// See the io implementation of this function.
+Future<void> processBrowserCommand(dynamic command) async {
+  throw UnimplementedError('processCommand is not used inside the browser');
```

Comment:
s/processCommand/processBrowserCommand/

Or maybe we shouldn't mention the name of the function to avoid typos altogether.